gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java

```diff
@@ -289,6 +286,22 @@ private void subscriberOnException(IOException exception) {
     incrementGcsExceptionCount();
   }
 
+  /**
+   * Updating the corresponding statistics
+   *
+   * @param strBuilder
+   */
+  @Subscribe
+  private void subscriberOnString(StringBuilder strBuilder) {
```

Comment:
Let us create a a new type in utils instead of using String type. Strong typing is better.